### DISCUSSION TITLE
Support for other JSX frameworks

### DISCUSSION THREAD

**Discussion by Mokshit06** [code-hike/codehike#305] [Ideas and feedback]
Right now code hike is limited to react, but since it uses mdx it should be possible to build adapters for other frameworks that support mdx/jsx too like solidjs.

Is this something that is on the roadmap?

**Comment by pomber** [comment]
Hi, see #90. React is a big dependency. A more framework-agnostic rewrite is possible but it's not on the roadmap right now.
